source/contracts/reporting/Market.sol

```diff
@@ -329,18 +329,28 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         return true;
     }
 
+    function withdrawInEmergency() public onlyInBadTimes onlyOwner returns (bool) {
+        if (this.balance > 0) {
+            require(msg.sender.call.value(this.balance)());
```

Comment:
We should put the ETH transfer after the REP transfer to avoid reentrancy issues.  I don't think its actually a problem here, but its generally a good practice.